release/rancher/rancher.go

```diff
@@ -71,3 +83,64 @@ func rancherImages(imagesURL string) (string, error) {
 	}
 	return string(images), nil
 }
+
+func CheckRancherDockerImage(tag string) error {
+	ctx := context.Background()
+	org := "rancher"
```

Comment:
I think we can make these constants and probably have them defined in another level up or somewhere else in the repo to be consumed here.